java/card-bot/src/main/java/com/google/hangouts/chat/card/Bot.java

```diff
@@ -0,0 +1,209 @@
+                        .setBottomLabel("The bottom label")
+                        .setIcon("STAR");
+                widgets.add(new WidgetMarkup().setKeyValue(widget));
+            } else if (s.contains("interactivetextbutton")) {
+                FormAction action = new FormAction()
+                        .setActionMethodName(INTERACTIVE_TEXT_BUTTON_ACTION)
+                        .setParameters(customParameters);
+                OnClick onClick = new OnClick().setAction(action);
+                TextButton button = new TextButton()
+                        .setText("INTERACTIVE BUTTON")
+                        .setOnClick(onClick);
+                Button widget = new Button().setTextButton(button);
+                widgets.add(new WidgetMarkup().setButtons(Collections.singletonList((widget))));
+            } else if (s.contains("interactiveimagebutton")) {
+                FormAction action = new FormAction()
+                        .setActionMethodName(INTERACTIVE_IMAGE_BUTTON_ACTION)
+                        .setParameters(customParameters);
+                OnClick onClick = new OnClick().setAction(action);
+                ImageButton button = new ImageButton()
+                        .setIcon("EVENT_SEAT")
+                        .setOnClick(onClick);
+                Button widget = new Button().setImageButton(button);
+                widgets.add(new WidgetMarkup().setButtons(Collections.singletonList((widget))));
+            } else if (s.contains("textbutton")) {
+                OpenLink openLink = new OpenLink().setUrl(REDIRECT_URL);
+                OnClick onClick = new OnClick().setOpenLink(openLink);
+                TextButton button = new TextButton()
+                        .setText("TEXT BUTTON")
+                        .setOnClick(onClick);
+                Button widget = new Button().setTextButton(button);
+                widgets.add(new WidgetMarkup().setButtons(Collections.singletonList((widget))));
+            } else if (s.contains("imagebutton")) {
+                OpenLink openLink = new OpenLink().setUrl(REDIRECT_URL);
+                OnClick onClick = new OnClick().setOpenLink(openLink);
+                ImageButton button = new ImageButton()
+                        .setIcon("EVENT_SEAT")
+                        .setOnClick(onClick);
+                Button widget = new Button().setImageButton(button);
+                widgets.add(new WidgetMarkup().setButtons(Collections.singletonList((widget))));
+            } else if (s.contains("image")) {
+                Image widget = new Image().setImageUrl("https://goo.gl/Bpa3Y5");
+                widgets.add(new WidgetMarkup().setImage(widget));
+            }
+        }));
+
+        Section section = new Section()
+                .setWidgets(widgets);
+        card.setSections(Collections.singletonList(section));
+        return card;
+    }
+
+    /**
+     * Creates a responseStr for when the user clicks on an interactive card.
```

Comment:
let's remove what might be considered an implementation detail from the javadocs (specifically 'responseStr')